templates/default/jupyter-launch.sh.erb

```diff
@@ -52,7 +57,17 @@ if [ -f "$PID_FILE" ] ; then
     rm $PID_FILE
 fi
 
-docker run --rm -d --name $CONTAINER_NAME --cidfile=$PID_FILE\
+JUPYTER_HOPSFS_MOUNT=false
+DOCKER_FUSE_ARGS=""
+WORKING_DIR=$SECRET_DIR
+if [ -f ${JUPYTER_HOME}/conf/jupyter_contents_manager_config.py ]; then
+  JUPYTER_HOPSFS_MOUNT=true
+  paste ${JUPYTER_HOME}/conf/jupyter_contents_manager_config.py >> ${JUPYTER_HOME}/conf/jupyter_notebook_config.py
```

Comment:
No. Since we want to be able to able configure to use either of the two, I keep the configuration for jupyter_contents_manager in a separate template. When we are using the hdfs contents manager then we merge it (jupyter_contents_manager) with the rest of the configuration.

Reply:
Can you correctly template the config file from the beginning in Hopsworks? If the remote_fs_driver is hdfscontentsmanager then template the correct properties. I'm not really fond of modifying the file in two places.

Reply:
Do you mean to do it glassfish?

Reply:
Yes